src/WebJobs.Extensions.DurableTask/EventSourceListener.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class EventSourceListener : EventListener
+    {
+        private readonly LinuxAppServiceILogger logger;
+
+        public EventSourceListener(LinuxAppServiceILogger logger)
+        {
+            this.logger = logger;
+        }
+
+        protected override void OnEventSourceCreated(EventSource eventSource)
+        {
+            if (eventSource.Name == "DurableTask-Core" ||
+                eventSource.Name == "DurableTask-AzureStorage" ||
+                eventSource.Name == "WebJobs-Extensions-DurableTask")
```

Comment:
Should we consider accepting anything that has `DurableTask` in the name?  It would be convenient if this could continue working even for `DurableTask-SqlServer` and whatever the name of Sebastian's E1 backend is.

Reply:
Maybe! Could this end up catching customer-managed eventsources if they also implement an EventSource with "DurableTask" in its name? That's mostly what I was trying to avoid.

Reply:
Although I suppose that the linux-platform-level code is already filtering EventSources that it doesn't recognize, so maybe this is a non-concern. I was simply trying to be robust by being explicit about what we capture

Reply:
In that case, can you also add DurableTask-SqlServer?